orion/pkg/repos/session_repo.go

```diff
@@ -51,7 +53,8 @@ func (sr *sessionRepo) SelectAllByClassId(classId string) ([]domains.Session, er
 			&session.StartsAt,
 			&session.EndsAt,
 			&session.Canceled,
-			&session.Notes); errScan != nil {
+			&session.Notes,
+			&session.PublishedAt); errScan != nil {
```

Comment:
Since the `published_at` column is after the `deleted_at`, I think this should also be after `&session.DeletedAt` as well as for the other 2 domains.